### PR TITLE
Updating Browserslist Dependency caniuse-lite

### PR DESCRIPTION
**Pull request by Atharva321**
Signed-off-by: Atharva321 <atharvapatil19202@gmail.com>

#### What it does

Fixes: #127 
This PR updates the caniuse-lite browserslist dependency

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,5 +43,6 @@
   "workspaces": [
     "applications/*",
     "theia-extensions/*"
-  ]
+  ],
+  "dependencies": {}
```
Comment (vince-fugnitto) [inline package.json:47]: I understand that changes in this file were temporarily added by executing `npx browserslist@latest --update-db`, but we can now revert these updates now that the `yarn.lock` was upgraded. You can `amend` the pull-request and revert changes made to `package.json`.

Comment (Atharva321) [reply package.json:47]: ok

**Review by vince-fugnitto — COMMENTED** [review]
@Atharva321 would you be able to squash your changes into a single commit?

**Comment by Atharva321** [comment]
Yes Doing.

**Comment by vince-fugnitto** [comment]
@Atharva321 looks like it didn't quite work out (2 commits became 4 :smile:)
The goal would be to have a single commit with your changes, please let us know if you need any help.

**Comment by Atharva321** [comment]
Yes.How can I fix this?
And also when I squashed the first two commits into one then why they are still  there?

**Comment by vince-fugnitto** [comment]
> Yes.How can I fix this?
> And also when I squashed the first two commits into one then why they are still there?

@Atharva321 I updated it for your, I'm not really sure what you did prior in order to get the state you had.

To fix it I did:
- `git log`: and recorded the `sha` of your good commit
- `git reset --hard master`: reset the branch to master
- `git cherry-pick ${sha}`: cherry-picked the previous sha
- cleaned up the `package.json` again by performing `git checkout origin/master package.json` - resets the file to master

**Review by vince-fugnitto — APPROVED** [review]
I no longer see the warning in the build log: 
- https://github.com/eclipse-theia/theia-blueprint/pull/150/checks?check_run_id=3364249405#step:5:55

Thank you for your contribution @Atharva321 :+1:

**Comment by Atharva321** [comment]
Thx for fixing

I ran this following commands.
```
git reset --hard HEAD~2
git merge --squash HEAD@{1}
git commit
git push
```
after that 2 commits become 4
and was not aware about cherry-pick cmd

**Comment by vince-fugnitto** [comment]
@Atharva321 when you had two commits, my usual workflow is to do the following:

- `git rebase -i HEAD~2`
- and in the interactive rebase simply squash the commit that updated the `package.json`.

**Pull request merged**